File: lab8/miniSearch/src/app/model/athlete.ts
export class Athlete {

    firstName: string;
    secondName: string;
    age: number;
    dyscipline: string;
    club: string;
    nationality: string

    constructor(firstName: string, secondName: string, age: number, dyscipline: string, club: string, nationality: string) {
        this.firstName = firstName;
        this.secondName = secondName;
        this.age = age;
        this.dyscipline = dyscipline;
        this.club = club;
        this.nationality = nationality;
    }
}

File: lab9/flask-server/athlete.py
class Athlete:
    def __init__(self, id, firstName, secondName, age, disc, club, nationality):
        self.id = id
        self.firstName = firstName
        self.secondName = secondName
        self.age = age
        self.disc = disc
        self.club = club
        self.nationality = nationality

    def serialize(self):
        return {
            'id': self.id,
            'firstName': self.firstName,
            'secondName': self.secondName,
            'age': self.age,
            'disc': self.disc,
            'club': self.club,
            'nationality': self.nationality
        }